src/core/src/runtime/ov_tensor.cpp

```diff
@@ -34,12 +34,11 @@ Tensor::~Tensor() {
 
 Tensor::Tensor(const Tensor& tensor, const std::shared_ptr<void>& so) : _impl{tensor._impl}, _so{tensor._so} {
     OPENVINO_ASSERT(_impl != nullptr, "Tensor was not initialized.");
-    _so.emplace_back(so);
+    if (!_so)
+        _so = so;
```

Comment:
do we need this check?

Reply:
Yes because some meta plugins can add pointers (for example proxy), but we need to store only initial pointer.
Maybe I will remove this code after the refactoring of plugin API.